sdk/redisenterprise/Azure.ResourceManager.RedisEnterprise/api/Azure.ResourceManager.RedisEnterprise.netstandard2.0.cs

```diff
@@ -174,6 +221,35 @@ public partial class ImportRedisEnterpriseDatabaseContent
         public ImportRedisEnterpriseDatabaseContent(System.Collections.Generic.IEnumerable<System.Uri> sasUris) { }
         public System.Collections.Generic.IList<System.Uri> SasUris { get { throw null; } }
     }
+    public partial class LocationInfo
+    {
+        internal LocationInfo() { }
+        public System.Collections.Generic.IReadOnlyList<Azure.ResourceManager.RedisEnterprise.Models.Capability> Capabilities { get { throw null; } }
+        public Azure.Core.AzureLocation? Location { get { throw null; } }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct Name : System.IEquatable<Azure.ResourceManager.RedisEnterprise.Models.Name>
```

Comment:
Please rename this type to `RedisEnterpriseSkuDetailsName`

Reply:
Updated